### DISCUSSION TITLE
Do Merkury Innovations and Geeni Products Still Work?

### DISCUSSION THREAD

**Discussion by RYoder97** [jasonacox/tinytuya#547] [Q&A] [answered]
Recently I tried picking up Merkury bulbs from the local Walmart. While they would show up and pair with the Merkury app, they would not show up, let alone pair, in the Tuya Smart app.

I'm looking to get GU10 bulbs and candelabra bulbs from Geeni (so we can stop using adapters with full sized bulbs), but I just found out Geeni is owned by Merkury. Are all their recent devices now no longer Tuya compatible? I have a handful of older Geeni and Merkury devices that ARE on my Tuya app and am using with Tinytuya. Could a new firmware update in the future remove them from Tuya and force them to only pair with their respective app?

**Comment by uzlonewolf** [comment, marked as answer]
No idea about Merkury, but I picked up some Geeni's a few weeks ago and they worked fine.

**Reply by uzlonewolf** [reply]
It's theoretically possible they could break it with an update, but I've never seen it happen.  Firmware updates in general are really rare.   Usually whatever a device is released with is all that device will ever see.

**Reply by RYoder97** [reply]
Awesome, thanks for letting me know. I suppose I can just order the devices off of Amazon vs through mygeeni.com to get speedy shipping an a no-hassle return experience in case they don't work for whatever reason. And yeah, I kinda figured that was the case with devices and firmware updates. It's a rare instance where you're glad that an OEM is too lazy to be bothered to update things!
